### DISCUSSION TITLE
请问有计划支持一个协作平台下的多个“库”吗？例如语雀

### DISCUSSION THREAD

**Discussion by Uxspy** [LetTTGACO/elog#68] [Q&A] [answered]
RT，首先感谢elog的帮助，在此提一个问题：elog目前都是支持单个语雀知识库，可不可以支持多个语雀知识库

因为对于vitepress来说，只要把不同的知识库用目录分割，并且配置不同的sidebar和navi，它是可以支持的

**Comment by LetTTGACO** [comment, marked as answer]
可以的啊，文档中有提：[如何同步多个知识库/数据表](https://elog.1874.cool/notion/qa#%E5%A6%82%E4%BD%95%E5%90%8C%E6%AD%A5%E5%A4%9A%E4%B8%AA%E7%9F%A5%E8%AF%86%E5%BA%93-%E6%95%B0%E6%8D%AE%E8%A1%A8)

具体原理就是 运行多个配置文件，不同的知识库输出到统一/不同的文件夹
